### DISCUSSION TITLE
Have preview-tui support tmux 2.x

### DISCUSSION THREAD

**Discussion by alexisthual** [jarun/nnn#1636] [Ideas]
Good afternoon!

Thank you for this awesome software.

I was wondering if it'd be possible to add support for tmux 2.x in the `preview-tui` plugin. I'm using `nnn` extensively on machines where I am not a sudoer and whose os are a bit old (ubuntu 18, centos 8), so it's not a situation where I can bump `tmux`.

Thanks!

**Comment by jarun** [comment]
I don't think it's technically feasible.
tmux 3.0 is required for setting environment variables in a new pane.

**Reply by alexisthual** [reply]
Oh noo :(
Anyways, thank you for your answer!
